component/file_cache/file_cache.go

```diff
@@ -71,7 +71,6 @@ type FileCache struct {
 	missedChmodList sync.Map
 	mountPath       string
 	allowOther      bool
-	offloadIO       bool
```

Comment:
Why are we removing this option ?
By default we handle file read/write from libfuse layer itself and call does not come to file-cache. But if we want to debug any specific case with file-cache then we need these calls and this options allows you to disable the libfuse handler short-circuiting.